### DISCUSSION TITLE
Dynamically switch between windows/linux and Mac

### DISCUSSION THREAD

**Discussion by HalimNaim** [manna-harbour/miryoku#229] [Ideas]
I have a work mac and my personal is a linux. I would really like if there was an option to dynamically remap the copy/paste/undo/redo keys and so on between the 2 modes instead of having to select one at compile time. In my personal keymap I added a global boolean variable with a key to toggle the value. then depending on that value, the appropriate keycodes are sent for the OS, while keeping all the keys in the same place.

**Comment by natecox** [comment]
This would make me very happy as well. Mac for work, Linux for play.
